script.config.dist

```diff
@@ -1,6 +1,6 @@
 DEPLOY_DIR=/srv/http/msqur
 BACKUP_DIR=~/
-DB_HOST=localhost
-DB_USER=msqur
+DB_HOST=127.0.0.1
```

Comment:
localhost won't work with the mariadb docker container I was using

Reply:
No need to change this, machine-specific config goes into script.config.